packages/sirius-web/frontend/sirius-web-application/src/index.ts

```diff
@@ -62,6 +62,8 @@ export {
   navigationBarMenuHelpURLExtensionPoint,
   navigationBarMenuIconExtensionPoint,
 } from './navigationBar/NavigationBarMenuExtensionPoints';
+export { PublishLibraryDialog } from './omnibox/PublishLibraryDialog';
+export { type PublishLibraryDialogProps } from './omnibox/PublishLibraryDialog.types';
```

Comment:
The `PublishLibraryDialog` is meant to be reusable by any command that only requires a version and a description. This is why I exported it (we'll use it in SysON). This is also why it has `title` and `message` in its props.
I didn't export `usePublishLibraries` because I don't see a reason to right now, but it can be added if necessary.